spec/models/contact_spec.rb

```diff
@@ -0,0 +1,75 @@
+RSpec.describe Contact, type: :model do
+  let(:user) { create(:user) }
+
+  describe 'associations' do
+    it { should belong_to(:user) }
+    subject { build(:contact) }
+  end
+
+  # Validations
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_presence_of(:relationship) }
+    it { should validate_presence_of(:user_id) }
+
+    it { should allow_value('friend', 'family', 'colleague').for(:relationship) }
+    it { should_not allow_value('enemy').for(:relationship) }
+
+    it { should allow_value('example@example.com').for(:email) }
+    it { should_not allow_value('not-an-email').for(:email) }
+  end
+
+  # Attributes
+  context 'attribute validations' do
+    subject(:contact) do
+      build(:contact, 
+            name: 'John Doe', 
+            email: 'john@example.com', 
+            phone_number: '123-456-7890', 
+            address: '123 Some St, City', 
+            social_media_handle: '@johndoe',
+            user: user)
```

Comment:
This should just be build without attributes no? Or does the factory bot not work here for some reason🤖🔥